docs/manual/java/releases/Highlights15.md

```diff
@@ -0,0 +1,29 @@
+# What's new in Lagom 1.5?
+
+This page highlights the new features of Lagom 1.5. If you want to learn about the changes you need to make when you migrate to Lagom 1.5, check out the [[Lagom 1.5 Migration Guide|Migration15]].
+
+
+
+## TLS Support
+
+Lagom in both [[dev mode|ConfiguringServicesInDevelopment#Using-HTTPS-in-development-mode]] and [[tests|Test#How-to-use-TLS-on-tests]] supports basic usage of TLS by means of self-signed certificates provided by the framework.
+
+## Cluster Formation
+
+A new mechanism to form and [[join an Akka Cluster|Cluster#Joining]] is introduced in Lagom 1.5. Apart from the original `Manual Cluster Formation` the new `Akka Cluster Bootstrap` is now supported. This new mechanism is introduced with lower precedence than `Manual Cluster Formation` so if you rely on the use of a list of `seed-nodes` then everything will work as before. On the other hand, `Akka Cluster Bootstrap` takes precedence over the `join-self` cluster formation for single-node clusters. If you use single-node clusters via `join-self` you will have to explicitly disable `Akka Cluster Bootstrap`:
+
+```
+lagom.cluster.bootstrap.enabled = false
+```
+
+##  Additional Routers
+
+Starting in Lagom 1.5 it is possible to extend the routes exposed by your service. So not only your service will expose thhe calls listed on  `Service.Descriptor` but can also serve endpoints handled by `additionalRouters`. The [[documentation|AdditionalRouters]] covers all the details. Additional routers makes it trivial to extend a `Service.Descriptor` with features natively supported by Play such has Upload File. There's even [Lagom recipes](https://github.com/lagom/lagom-recipes/tree/master/file-upload) detailing such a use case.
```

Comment:
fix "...expose thhe calls..."

```suggestion
Starting in Lagom 1.5 it is possible to extend the routes exposed by your service. So not only your service will expose the calls listed on  `Service.Descriptor` but can also serve endpoints handled by `additionalRouters`. The [[documentation|AdditionalRouters]] covers all the details. Additional routers makes it trivial to extend a `Service.Descriptor` with features natively supported by Play such as File Upload. There's even [Lagom recipes](https://github.com/lagom/lagom-recipes/tree/master/file-upload) detailing such a use case.
```

Reply:
Shall we merge 
https://github.com/lagom/lagom-recipes/pull/67 and https://github.com/lagom/lagom-recipes/pull/68?

Reply:
Merged

Reply:
The change suggested by @dwijnand above superseded this suggestion.

Reply:
No entirely. We should say: "File Upload", not "Upload File".

Reply:
Alternatively, we could write: "with features natively supported by Play such as uploading a file"